ansible/roles/apache-flink/templates/flink-init.j2

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# The path where Apache Flink is installed.
+INSTALLATION_PATH="{{ installation_path }}"
+
+# The full path of the pid file to use.
+PIDFILE="$INSTALLATION_PATH/flink/flink.pid"
+
+# The full path of the lock file to use.
+LOCKFILE="$INSTALLATION_PATH/flink/flink-lock"
+
+# The command that will start Apache Flink.
+START_COMMAND="$INSTALLATION_PATH/flink/bin/yarn-session.sh -n {{ number_of_taskmanagers }} -tm {{ ram_per_task_manager }}"
+
+# The command that will stop Apache Flink. Note that, HADOOP_HOME home variable should be set outside from this script and
+# before Flink's deployment.
+STOP_COMMAND="$HADOOP_HOME/bin/yarn application --kill"
```

Comment:
HADOOP_HOME is an environment variable and a prerequisite for Apache Flink installation. It should be set up before Apache Flink is installed. At the current moment, it is defined within /etc/environment file by the Ansible common role[1].

https://github.com/grnet/okeanos-LoD/blob/devel/ansible/roles/common/files/environment